packages/taquito-signer/src/derivation-tools/ed25519.ts

```diff
@@ -4,7 +4,8 @@ import { SHA512 } from '@stablelib/sha512';
 import { generateKeyPairFromSeed } from '@stablelib/ed25519';
 import { ExtendedPrivateKey, Hard } from './index';
 import { parseHex } from './utils';
-import { InvalidDerivationPathError, InvalidSeedLengthError } from '../errors';
+import { InvalidSeedLengthError } from '../errors';
+import { InvalidDerivationPathError } from '../errors';
```

Comment:
not sure why they're on different lines, they should be from the same file it seems